tornado/ioloop.py

```diff
@@ -43,8 +43,9 @@
 import time
 import traceback
 import math
+from multiprocessing import cpu_count
```

Comment:
Looking at the Python documentation, it says `cpu_count` may raise a `NotImplementedError` without explaining when or why. Since the function in `tornado.process` defaults to 1 if all else fails, I agree it makes more sense to use that.